bin/build.js

```diff
@@ -109,7 +112,10 @@ projects.forEach( async ( file ) => {
 
 		// We `await` because JS needs to be built first— the first webpack step deletes the build
 		// directory, and could remove the built CSS if it was truely async.
-		await maybeBuildJavaScript( path.resolve( path.join( basePath, 'src' ) ), outputDir );
+		await maybeBuildBlock( path.resolve( path.join( basePath, 'src' ) ), outputDir );
+
+		// `maybeBuildBlock` supports SCSS, but current blocks still have postCSS
+		// files. Until those are converted, we still need a separate PostCSS step.
```

Comment:
So will we eventually phase out all PostCSS in favor of SCSS? I'm not sure if the discussion [here](https://github.com/WordPress/wordcamp.org/pull/1172#discussion_r1425590556) is relevant because it seems to be discussing the opposite idea.

Reply:
Oh, thanks for pointing me to that conversation, it happened while I was away :)

Honestly I was remembering [this recent bug](https://github.com/WordPress/wporg-mu-plugins/pull/554#discussion_r1464064815) where the CSS was not building properly because it was using Sass formatting. I didn't realize there had been a more recent discussion than [the 2022 issue](https://github.com/WordPress/wporg-parent-2021/issues/1).

… turns out though, [wp-scripts does support `*.pcss`](https://github.com/WordPress/gutenberg/blob/2569121b896ec18db4ce4262d6bbfbac72b32045/packages/scripts/config/webpack.config.js#L211C1-L214C6), so we could remove this without forcing a Sass decision. I'll update the comment here and try that in a new PR.